src/media/js/tracking_events.js

```diff
@@ -27,6 +29,11 @@ define('tracking_events',
     var FEATURED_APP_VALUE = 999999999;
     var EDITORIAL_BRAND_VALUE = 999999998;
 
+    // Track region.
+    consumer_info.promise.done(function() {
+        setVar(11, user_helpers.region());
```

Comment:
I'd consider removing the line from `mobilenetwork.js`. `consumer_info` default promise should always be resolved, so you'll always get this, so you don't need to do it in `mobilenetwork.js` as well.